amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseTaker.java

```diff
@@ -191,9 +191,9 @@ synchronized Map<String, Lease> takeLeases(Callable<Long> timeProvider)
                 return takenLeases;
             }
 
-            List<Lease> expiredLeases = getExpiredLeases();
+            List<Lease> availableLeases = getAvailableLeases();
```

Comment:
Changes in this file are all related to changing `expired` lease concept for `available` lease concept.